commands.py

```diff
@@ -0,0 +1,48 @@
+from .protocol import Call, CallsDirection, CallsRequestParams, CallsResponse
+from LSP.plugin import Request
+from LSP.plugin import Session
+from LSP.plugin.core.protocol import LocationLink
+from LSP.plugin.core.registry import LspTextCommand
+from LSP.plugin.core.typing import Optional
+from LSP.plugin.core.views import text_document_position_params
+from LSP.plugin.locationpicker import LocationPicker
+import functools
+import sublime
+
+
+SESSION_NAME = "LSP-typescript"
+
+
+class LspTypescriptCallsCommand(LspTextCommand):
+
+    session_name = SESSION_NAME
+
+    def is_enabled(self) -> bool:
+        selection = self.view.sel()
+        return len(selection) > 0 and super().is_enabled()
+
+    def run(self, edit: sublime.Edit, direction: CallsDirection) -> None:
+        session = self.session_by_name(self.session_name)
+        if session is None:
+            return
+        position_params = text_document_position_params(self.view, self.view.sel()[0].b)
+        params = {
+            'textDocument': position_params['textDocument'],
+            'position': position_params['position'],
+            'direction': direction
+        }  # type: CallsRequestParams
+        session.send_request(Request("textDocument/calls", params), functools.partial(self.on_result_async, session))
+
+    def on_result_async(self, session: Session, result: Optional[CallsResponse]) -> None:
+        if not result:
+            return
+
+        def to_location_link(call: Call) -> LocationLink:
+            return {
+                'targetUri': call['location']['uri'],
+                'targetSelectionRange': call['location']['range'],
+            }
+
+        locations = list(map(to_location_link, result['calls']))
+        self.view.run_command("add_jump_record", {"selection": [(r.a, r.b) for r in self.view.sel()]})
```

Comment:
Is this line necessary?
I deleted this line locally and I do not see a difference in behavior `jump_back` or `jump_forward`.

Reply:
Not sure since this is copied from equivalent implementations for "find references" in LSP.

Reply:
Maybe @rwols knows why this line exist 
and maybe if it can be removed? :)

@rchl just so you know, 
this is not a blocker for this PR to get merged. we can discuss this after we merge this.